site/docs/components/App.tsx

```diff
@@ -23,31 +23,10 @@ const wagmiConfig = createConfig({
   },
 });
 
-export default function App({ children }: { children: ReactNode }) {
-  const [theme, setTheme] = useState('light');
```

Comment:
Curious why we are removing theming this level?

Reply:
This breaks components updating based on dark/light mode, see [/swap/swap](https://onchainkit-czk9bf4hw-coinbase-vercel.vercel.app/swap/swap) as an example

This component should be using the light mode theme:

<img width="600" alt="Screenshot 2024-10-24 at 5 27 52 PM" src="https://github.com/user-attachments/assets/b666efbf-b2a3-4de4-8c88-b513277e54fd">

Reply:
I see, sorry about that. 

The reason I pulled `theme` up to ComponentPreview` level is that I need theme at that level in order to render the code block with the right style. So I thought that I would keep it at that level and just pass as props to `<App>` which lives in `ActiveComponent`s. 

I didn't realize that `<App>` was used elsewhere and had fixed the structure for the files I could find. I see now that it's also used in markdown files. 

Will get this fixed.